lib/tasks/set_language_model_for_alegre_team_bot_installation.rake

```diff
@@ -0,0 +1,13 @@
+namespace :check do
+  # bundle exec rake check:fix_untitled_sources['id:1&name:sname']
+  # bundle exec rake check:fix_untitled_sources to fix sources with name == 'Untitled'
+  desc "Set Alegre Team Bot Installation model in use for generating string similarities (usage: bundle exec rake check:set_language_model_for_alegre_team_bot_installation['team_id:1','model_name:indian-sbert'])"
+  task set_language_model_for_alegre_team_bot_installation: :environment do |_t, args|
+    team = Team.find(args.team_id)
+    tbi = team.team_bot_installations.select{|x| x.user.login == "alegre"}.first
+    settings = tbi.json_settings
+    settings["alegre_model_in_use"] = args.model_name
+    tbi.json_settings = settings
+    tbi.save!
```

Comment:
We have a `lib/check_settings.rb` library to abstract this kind of operation, so it can simply be:
```
tbi.set_alegre_model_in_use = args.model_name
tbi.save!
```